### PR TITLE
Header Auth requires_token decorator

### PR DESCRIPTION
**Pull request by FerrySchuller**
Added header authentication in the requires_token decorator.

### CODE REVIEW

File: eve_auth_jwt/auth.py [hunk 1]
```diff
@@ -162,7 +162,10 @@ def requires_token(audiences=[], allowed_roles=[]):
     def requires_token_wrapper(f):
         @wraps(f)
         def decorated(*args, **kwargs):
-            token = request.args.get('access_token')
+            if request.args.get('access_token'):
+                token = request.args.get('access_token')
+            else:
+                token = request.headers.get('Authorization').split(' ')[1]
```
Comment (rs) [inline eve_auth_jwt/auth.py]: This might throw an "index out of range" exception.

File: eve_auth_jwt/auth.py [hunk 2]
```diff
@@ -162,7 +162,10 @@ def requires_token(audiences=[], allowed_roles=[]):
     def requires_token_wrapper(f):
         @wraps(f)
         def decorated(*args, **kwargs):
-            token = request.args.get('access_token')
+            if request.args.get('access_token'):
+                token = request.args.get('access_token')
```
Comment (rs) [inline eve_auth_jwt/auth.py]: A more idiomatic way would be to catch the key access error.

**Comment by eventh** [comment]
I needed this functionality myself, so I made appropriate changes in https://github.com/eventh/eve-auth-jwt/commit/22e7aa3505002475a54d6deffe1d43272ed1103e

**Comment by rs** [comment]
Fixed by #11

**Pull request closed without merging**